source/support.tex

```diff
@@ -530,251 +530,141 @@
-  \tcode{<compare>} \\ \rowsep
-\defnlibxname{cpp_lib_string_udls}                          & \tcode{201304L} &
-  \tcode{<string>} \\ \rowsep
-\defnlibxname{cpp_lib_string_view}                          & \tcode{201606L} &
-  \tcode{<string>} \tcode{<string_view>} \\ \rowsep
-\defnlibxname{cpp_lib_three_way_comparison}                 & \tcode{201711L} &
-  \tcode{<compare>} \\ \rowsep
-\defnlibxname{cpp_lib_to_array}                             & \tcode{201907L} &
-  \tcode{<array>} \\ \rowsep
-\defnlibxname{cpp_lib_to_chars}                             & \tcode{201611L} &
-  \tcode{<charconv>} \\ \rowsep
-\defnlibxname{cpp_lib_transformation_trait_aliases}         & \tcode{201304L} &
-  \tcode{<type_traits>} \\ \rowsep
-\defnlibxname{cpp_lib_transparent_operators}                & \tcode{201510L} &
-  \tcode{<memory>} \tcode{<functional>} \\ \rowsep
-\defnlibxname{cpp_lib_tuple_element_t}                      & \tcode{201402L} &
-  \tcode{<tuple>} \\ \rowsep
-\defnlibxname{cpp_lib_tuples_by_type}                       & \tcode{201304L} &
-  \tcode{<utility>} \tcode{<tuple>} \\ \rowsep
-\defnlibxname{cpp_lib_type_trait_variable_templates}        & \tcode{201510L} &
-  \tcode{<type_traits>} \\ \rowsep
-\defnlibxname{cpp_lib_uncaught_exceptions}                  & \tcode{201411L} &
-  \tcode{<exception>} \\ \rowsep
-\defnlibxname{cpp_lib_unordered_map_try_emplace}            & \tcode{201411L} &
-  \tcode{<unordered_map>} \\ \rowsep
-\defnlibxname{cpp_lib_variant}                              & \tcode{201606L} &
-  \tcode{<variant>} \\ \rowsep
-\defnlibxname{cpp_lib_void_t}                               & \tcode{201411L} &
-  \tcode{<type_traits>} \\ \rowsep
-\end{LongTable}
+\begin{codeblock}
+#define @\defnlibxname{cpp_lib_addressof_constexpr}@               201603L // also in \libheader{memory}
+#define @\defnlibxname{cpp_lib_allocator_traits_is_always_equal}@  201411L
+  // also in \libheader{memory}, \libheader{scoped_allocator}, \libheader{string}, \libheader{deque}, \libheader{forward_list}, \libheader{list}, \libheader{vector},
+  // \libheader{map}, \libheader{set}, \libheader{unordered_map}, \libheader{unordered_set}
+#define @\defnlibxname{cpp_lib_any}@                               201606L // also in \libheader{any}
+#define @\defnlibxname{cpp_lib_apply}@                             201603L // also in \libheader{tuple}
+#define @\defnlibxname{cpp_lib_array_constexpr}@                   201603L // also in \libheader{iterator}, \libheader{array}
+#define @\defnlibxname{cpp_lib_as_const}@                          201510L // also in \libheader{utility}
+#define @\defnlibxname{cpp_lib_atomic_flag_test}@                  201907L // also in \libheader{atomic}
+#define @\defnlibxname{cpp_lib_atomic_is_always_lock_free}@        201603L // also in \libheader{atomic}
+#define @\defnlibxname{cpp_lib_atomic_lock_free_type_aliases}@     201907L // also in\libheader{atomic}
```

Comment:
`// also in\libheader{atomic}` is missing a space between "in" and `\libheader`.

Reply:
Fixed.